backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/Derivation.kt

```diff
@@ -38,6 +38,37 @@ interface Derivation {
         numberOfAccounts: Int
     ): Array<String>
 
+    /**
+     * Derives a ZIP 32 Arbitrary Key from the given seed at the "wallet level", i.e.
+     * directly from the seed with no ZIP 32 path applied.
+     *
+     * The resulting key will be the same across all networks (Zcash mainnet, Zcash
+     * testnet, OtherCoin mainnet, and so on). You can think of it as a context-specific
+     * seed fingerprint that can be used as (static) key material.
+     *
+     * @param contextString a globally-unique non-empty sequence of at most 252 bytes that
+     *        identifies the desired context.
```

Comment:
If failing to meet this requirement can cause a panic, it must also be documented at this layer.

Reply:
A Rust panic becomes a `RuntimeException` on the Android side, so there's no panic exposed in the public API.